app/Http/Controllers/BarbersController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Barber;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Gate;
+
+class BarbersController extends Controller
+{
+    public function store(Request $request)
```

Comment:
Since `register` is really just the same as the `create` form, you may want to move it above `store` and rename it `create` to make it more like the traditional REST-ish Laravel controller shape.